lua/telescope/actions/init.lua

```diff
@@ -143,6 +143,87 @@ function actions.toggle_selection(prompt_bufnr)
   current_picker:toggle_selection(current_picker:get_selection_row())
 end
 
+--- Apply `f` to entries of current picker and returns list of mapped entries
+--- `f` takes (entry, index, row) as viable arguments in order
+---@param prompt_bufnr number: The prompt bufnr
+---@param f function: function to apply on entries of picker
+---@return table: result from mapped entries
+function actions.map_entries(prompt_bufnr, f)
```

Comment:
`actions` that have more than one parameter are usually prefixed with `_` because they are not "mapable". But this function makes even more sense in `actions.state` or a new `actions.util`

Reply:
Makes sense, any preference on `actions.state` or `actions.util`? Also makes prefixing with `_`  obsolete which I'd prefer.

Reply:
i prefer `actions/utils.lua`, make sure you add the file to `gendocs.lua` and write `@tag` and `@brief`